Fragment transformation

rachelbock opened the pull request:
Converted main activities to fragments and created a userData class and local database.
- Transformed all of the activities into fragments.
- Added the action bar directly to the Home Page activity and removed it's fragment.
- Added a userData class.
- Added a SQLiteHelper class to create a local databse to store the user data.

Rearrangement of project to include login activity. 
- Created a login activity to create and enter a user name.
- Created a local database to store user information from the user dialog fragment.
- Determined that it would be best to store data on external database.

Login Activity updated to be activity opened on launch of application. 
-Updated manifest to have Login Activity be main activity.
-On Submit Clicked buttons open home page.
-Got rid of writable database from Dialog Fragment. Will use External Database.

Commit:
Converted main activities to fragments and created a userData class and local database.

- Transformed all of the activities into fragments.
- Added the action bar directly to the Home Page activity and removed it's fragment.
- Added a userData class.
- Added a SQLiteHelper class to create a local databse to store the user data.

Commit:
Rearrangement of project to include login activity.

- Created a login activity to create and enter a user name.
- Created a local database to store user information from the user dialog fragment.
- Determined that it would be best to store data on external database.

Commit:
Login Activity updated to be activity opened on launch of application.

-Updated manifest to have Login Activity be main activity.
-On Submit Clicked buttons open home page.
-Got rid of writable database from Dialog Fragment. Will use External Database.

nonvirtualthunk commented on app/src/main/java/com/rage/clamber/Fragments/UserInfoDialogFragment.java:
```diff
@@ -20,13 +22,14 @@
-    @Bind(R.id.user_info_dialog_fragment_height_edit_text)
-    EditText heightEditText;
+    @Bind(R.id.user_info_dialog_fragment_height_ft_edit_text)
+    EditText heightFtEditText;
```
Go ahead and give all these fields an access modifier while you're in here

rachelbock replied:
will do - I think those are ones that I had but then reset everything!

nonvirtualthunk commented on app/src/main/java/com/rage/clamber/Fragments/WallsFragment.java:
```diff
@@ -0,0 +1,93 @@
+import butterknife.OnClick;
+
+/**
+ *
```
Javadoc

rachelbock replied:
Yep

nonvirtualthunk commented on app/src/main/java/com/rage/clamber/Fragments/UserInfoFragment.java:
```diff
@@ -0,0 +1,59 @@
+import butterknife.OnClick;
+
+/**
+ * A simple {@link Fragment} subclass.
```
Add a bit of info about what this fragment's purpose is

rachelbock replied:
will do

Commit:
Updates based off of pull request
- Added comments to new fragment classes.
- Added access modifiers to user dialog fragment resources.
